src/lib/hofs/functions.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * This method creates a function that invokes func once it’s called n or more times.
+ * @see https://youmightnotneed.com/lodash#after
```

Comment:
lodash has tree-shaking problems when using some methods, and lodash-es is just an automatically generated version of lodash based on the same code.

I've found that creating some higher order functions (hof) in codebases generally results in a better experience because we have full control over a smaller number of functions (I don't think anyone uses all of lodash in any single package).

I've never needed to create a hofs library though we probably should for our ipfs packages.